portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -29,16 +35,22 @@
 
   private ArrayList<Comment> comments;
   
-  @Override
-  public void init() {
-    comments = new ArrayList<Comment>();
-  }
-  
   /* responds with a JSON string containing authors and comments*/
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Convert the ArrayLists to JSON
+    // Query datastore for all comments
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Could this be a member variable? Or does it need to be instantiated every call?

Reply:
Changed.